commands/user.bee.inc

```diff
@@ -98,6 +187,180 @@ function users_bee_callback($arguments, $options) {
   );
 }
 
+/**
+ * Command callback: Create a user.
+ */
+function user_bee_create_callback($arguments, $options) {
```

Comment:
As above, should be:
```suggestion
function user_create_bee_callback($arguments, $options) {
```

Also for this function, note again the comments above about matching, where we can, the balance of arguments and options with the corresponding drush function.